release-team/role-handbooks/docs/README.md

```diff
@@ -186,22 +186,55 @@ Find .5-1 hour of time to meet with shadows and explain the release process. Wal
 
 1. Add contacts to the shadows release docs, e.g: [https://bit.ly/k8s114-contacts](https://docs.google.com/spreadsheets/d/1BiGSLuCqjglQS1bJvpKk6rKFMciebPkUndzgDRnJsns/edit?ts=5c3bd42a#gid=0)
 
+#### Ensure access is set yp
+
 1. Make sure all shadows have edit access to the enhancement spreadsheet.
 
 1. As a lead, make sure you are part of the [milestone-maintainers](https://github.com/orgs/kubernetes/teams/milestone-maintainers) and [sig-docs-en-owners](https://github.com/orgs/kubernetes/teams/sig-docs-en-owners).
 
 1. You need push access to the Kubernetes website repo (contact a SIG Docs chair if you don't have it)
 
-    ⚠️ Open the [integration branch] by creating a pull request against `master` referencing the `dev-[future release]` branch. **This should be done by the Docs Lead**, e.g: the [Release 1.14](https://github.com/kubernetes/website/pull/13174) PR uses the branch `dev-1.14`.
+   ⚠️ Open the [integration branch] by creating a pull request against `master` referencing the `dev-[future release]` branch. **This should be done by the Docs Lead**, e.g: the [Release 1.14](https://github.com/kubernetes/website/pull/13174) PR uses the branch `dev-1.14`.
 
-    This release pull request (also known as the [integration branch]) serves as the base for individual, component enhancement PRs of the release. A [integration branch] lets you bundle and merge multiple PRs simultaneously.
+   This release pull request (also known as the [integration branch]) serves as the base for individual, component enhancement PRs of the release. A [integration branch] lets you bundle and merge multiple PRs simultaneously.
 
-    ⚠️ Add the label `do-not-merge/hold` to the PR.
+   ⚠️ Add the label `do-not-merge/hold` to the PR.
 
 1. First PR in `dev-[future release]`: Update config.toml to show `[future release]` as the current version and add the `[future release]` entry to the drop-down, e.g: [config.toml diff](https://github.com/kubernetes/website/commit/851ef58fa8413e47bb18a15de53d9556be8c53dd#diff-618063036395fe9ee107f22b46c9eade)
 
     The intent is that your new branch should be showing as the current version IN the new branch...
 
+#### Update the website configuration ahead of the release
+
+Update the main `config.toml` based on the version in _master_, ready for the pending release. PR those changes it against `dev-[future release]` branch (which should already exist - this is a good check!)
+
+```shell
+# Step 1
+# Do this on a fresh local clone OF YOUR FORK
+# It's OK to use SSH for the git URL if you know how to do that
+git clone https://github.com/yourGitHubUsername/website.git kubernetes-website
+cd kubernetes-website
+# Step 2
```

Comment:
Nit: add spaces between the commands and the comments. Same with below. :arrow_down: